modular_skyrat/modules/medical/code/wounds/synth/blunt/secures_internals.dm

```diff
@@ -0,0 +1,388 @@
+			return TRUE
+
+	if (ready_to_secure_internals)
+		if (item_can_secure_internals(potential_treater))
+			return TRUE
+
+	return ..()
+
+/datum/wound/blunt/robotic/secures_internals/treat(obj/item/potential_treater, mob/user)
+	if (ready_to_secure_internals)
+		if (istype(potential_treater, /obj/item/stack/medical/bone_gel))
+			return apply_gel(potential_treater, user)
+		else if (!crowbarred_open && potential_treater.tool_behaviour == TOOL_CROWBAR)
+			return crowbar_open(potential_treater, user)
+		else if (item_can_secure_internals(potential_treater))
+			return secure_internals_normally(potential_treater, user)
+	else if (ready_to_resolder && (potential_treater.tool_behaviour == TOOL_WELDER) || (potential_treater.tool_behaviour == TOOL_CAUTERY))
+		return resolder(potential_treater, user)
+
+	return ..()
+
+/// Returns TRUE if the item can be used in our 1st step (2nd if T3) of repairs.
+/datum/wound/blunt/robotic/secures_internals/proc/item_can_secure_internals(obj/item/potential_treater)
+	return (potential_treater.tool_behaviour == TOOL_SCREWDRIVER || potential_treater.tool_behaviour == TOOL_WRENCH || istype(potential_treater, /obj/item/stack/medical/bone_gel))
+
+#define CROWBAR_OPEN_SELF_TEND_DELAY_MULT 2
+#define CROWBAR_OPEN_KNOWS_ROBO_WIRES_DELAY_MULT 0.5
+#define CROWBAR_OPEN_KNOWS_ENGI_WIRES_DELAY_MULT 0.5
+#define CROWBAR_OPEN_HAS_DIAG_HUD_DELAY_MULT 0.5
+#define CROWBAR_OPEN_WOUND_SCANNED_DELAY_MULT 0.5
+/// If our limb is essential, damage dealt to it by tearing it open will be multiplied against this.
+#define CROWBAR_OPEN_ESSENTIAL_LIMB_DAMAGE_MULT 1.5
+
+/// The "power" put into electrocute_act whenever someone gets shocked when they crowbar open our limb
+#define CROWBAR_OPEN_SHOCK_POWER 20
+/// The brute damage done to this limb (doubled on essential limbs) when it is crowbarred open
+#define CROWBAR_OPEN_BRUTE_DAMAGE 20
+
+/**
+ * Available during the "secure internals" step of T2 and T3. Requires a crowbar. Low-quality ghetto option.
+ *
+ * Tears open the limb, exposing internals. This massively increases the chance of secure internals succeeding, and removes the self-tend malice.
+ *
+ * Deals significant damage to the limb, and shocks the user (causing failure) if victim is alive, this limb is wired, and user is not insulated.
+ */
+/datum/wound/blunt/robotic/secures_internals/proc/crowbar_open(obj/item/crowbarring_item, mob/living/user)
+	if (!crowbarring_item.tool_start_check())
+		return TRUE
+
+	var/delay_mult = 1
+	if (user == victim)
+		delay_mult *= CROWBAR_OPEN_SELF_TEND_DELAY_MULT
+
+	var/knows_wires = FALSE
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
+		delay_mult *= CROWBAR_OPEN_KNOWS_ROBO_WIRES_DELAY_MULT
+		knows_wires = TRUE
+	else if (HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+		delay_mult *= CROWBAR_OPEN_KNOWS_ENGI_WIRES_DELAY_MULT
+		knows_wires = TRUE
+	if (HAS_TRAIT(user, TRAIT_DIAGNOSTIC_HUD))
+		if (knows_wires)
+			delay_mult *= (CROWBAR_OPEN_HAS_DIAG_HUD_DELAY_MULT * 1.5)
```

Comment:
Didn't you usually only have `1.25` for these normally? Won't it compound into something even higher?

Reply:
The default delay mult for having a diag hud here is 0.5

0.5 * 1.5 = 0.75

With delay mults, the lower is better, since youre directly multiplying it against the time it takes to do a thing

Reply:
Oh, right, my bad